teaser/project.py

```diff
@@ -1079,6 +1079,15 @@ def name(self):
 
     @name.setter
     def name(self, value):
-
-        self._name = value
-        self.modelica_project = value
+        if isinstance(value, str):
+            regex = re.compile('[^a-zA-z0-9]')
+            self._name = regex.sub('', value)
```

Comment:
On this place we need to change also the value of `self.modelica_project` 

Please also check why there is such an attribute at all, do we really need this?